### ISSUE TITLE
deeplab predict a blank segmentation masks

### ISSUE THREAD

**Issue by hakS07** [DrSleep/tensorflow-deeplab-lfov#23]
@DrSleep  Hi sir, I wish you can help me with this,
### System information
What is the top-level directory of the model you are using:
https://github.com/tensorflow/models/tree/master/research/deeplab
** Have I written custom code (as opposed to using a stock example script provided in TensorFlow):**
No
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):**ubuntu 18.04
**TensorFlow installed from (source or binary):**binary
**TensorFlow version (use command below):**1.13.1 CPU
**Bazel version (if compiling from source):**N/A
**CUDA/cuDNN version:**9.2/7.1
**GPU model and memory:**GeForce GTX 970M

Exact command to reproduce:
`python  train.py \ --logtostderr \ --vis_split = "train" \ --model_variant = "xception_65" \ --atrous_rates = 6 \ --atrous rates = 12 \ --atrous rates = 18 \ --output_stride = 16 \ --decoder_output_stride = 4 \ --training_number_of_steps = 1000 --train_crop_size = 513 \ --train_batch_size = 1 \ --train_crop_size = 513 \ --Fine_tune_batch_norm=False \ --Tf_initial_checkpoint = "./ Data / Init_models / Deelabv3_pascal_train_aug \ model.ckpt" --Initialize_last_layer = False \ --Last_layers_contain_logits_only = True \  --train_logdir="./data/log/train" \  --dataset_dir="./data/tfrecord" \  --dataset="pascal_voc_seg"`
### Describe the problem
Hi i trained deeplab model and does not predict anything just a black background, i don't know what's the problem
-training on custom dataset (iris data)
-data = RGB image + ( 0-1) label: 400 * 300
-classe=2
-convert to record format:
-training step:
==> Loss = 0.2 ～ 0.1
-convert to .pb step
python export_model.py \ --logtostderr \ -model_variant = "xception_65" \ --atrous_rates = 6 \ --atrous_rates = 12 \ --atrous_rates = 18 \ --output_stride = 16
until this step everything looks fine
-test step
![Capture](https://user-images.githubusercontent.com/34586193/59199118-14f8a580-8b8d-11e9-82fb-74f4ec0585fb.JPG)
![1](https://user-images.githubusercontent.com/34586193/59199124-188c2c80-8b8d-11e9-9ef5-229ad9aec606.JPG)
![2](https://user-images.githubusercontent.com/34586193/59199129-1c1fb380-8b8d-11e9-8707-57762aa4167f.JPG)
![3](https://user-images.githubusercontent.com/34586193/59199134-1fb33a80-8b8d-11e9-9963-930e18d48641.JPG)
